app.js

```diff
@@ -2,12 +2,23 @@ require('dotenv').config();
 var express = require('express');
 var port = process.env.SERVER_PORT;
 const { getAbsenceData } = require('./utils/analyzeAbsences');
+const { students, absenceRecords } = require('./db/fakeStudentData');
+const bodyParser = require('body-parser');
 
 var app = express();
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
 app.use(express.static('public'));
+
 app.get('/absences', async function(req, res) {
-  let absences = await getAbsenceData();
-  res.send(absences);
+  // let absences = await getAbsenceData();
+  res.send(absenceRecords);
```

Comment:
I notice this is the dev version of your code.  I'm merging with the assumption that you are sharing this with Emily for dev efforts.  If not, it needs to be corrected.